src/java/org/apache/cassandra/service/AutoRepairService.java

```diff
@@ -202,6 +202,16 @@ public void setParallelRepairCount(String repairType, int count)
         config.setParallelRepairCount(RepairType.parse(repairType), count);
     }
 
+    public void setAllowParallelReplicaRepair(String repairType, boolean enabled)
```

Comment:
NIT add missing override annotations :
```suggestion
    @Override
    public void setAllowParallelReplicaRepair(String repairType, boolean enabled)
```